src/app/components/arlas-wui-root/arlas-wui-root.component.ts

```diff
@@ -792,6 +798,15 @@ export class ArlasWuiRootComponent implements OnInit, AfterViewInit, OnDestroy {
     this.mapSettings.openDialog(this.mapSettingsService);
   }
 
+  private retrieveOpenedGridItem(){
+    // wrong place because fire before result list opened
```

Comment:
Does that mean that the method is not useful ?

Reply:
no its an old comment. I ll remove it